packages/shared/src/components/Feed.tsx

```diff
@@ -174,8 +174,7 @@ export default function Feed<T>({
       currentSettings.adSpot,
       numCards,
       showOnlyUnreadPosts,
-      query,
-      variables,
+      { query, variables, options: { refetchOnMount: true } },
```

Comment:
Is this not a default?

Reply:
I was also surprised to see this actually. Probably the mechanism for real-time updates in our feed is different, I could be wrong tho.

![image](https://user-images.githubusercontent.com/13744167/217021592-9dae8183-5658-43d6-b005-63124fb252fc.png)